src/bluesearch/embedding_models.py

```diff
@@ -272,8 +272,8 @@ def compute_database_embeddings(connection, model, indices, batch_size=10):
     sentences = retrieve_sentences_from_sentence_ids(indices, connection)
     n_sentences = len(sentences)
 
-    all_embeddings = list()
-    all_ids = list()
+    all_embeddings = []
+    all_ids = []
```

Comment:
What is the reasoning here? It seems like this is a reoccurring patter: `[]` instead of `list()`, `{}` instead of `dict()`,...

Reply:
Apparenty slightly faster and slightly safer as per [here](https://stackoverflow.com/a/6610783/2804645) and [here](https://softwareengineering.stackexchange.com/a/319549/109896).

To be honest, I'm not completely sold on the speed and safety gains. Probably best to view it as a convention to be used throughout the codebase.